tests/buildtest.cmd

```diff
@@ -106,11 +106,9 @@ set "__CMakeBinDir=%__CMakeBinDir:\=/%"
 if not defined __CleanBuild goto SkipCleanBuild
 echo %__MsgPrefix%Doing a clean test build
 
-:: MSBuild projects would need a rebuild
-set __msbuildCleanBuildArgs=/t:rebuild
```

Comment:
"rebuild" cannot be simply added to runtest.proj since currently it is used from buildtest.cmd as well as runtest.cmd. Ideally we might want to remove such dependency on runtest.proj when building test. Instead, I skipped passing /t:rebuild since we explicitly targets "/t:CreateTestOverlay" when building tests. Updated the change accordingly.